changelog/extern_cpp_overhaul.dd

```diff
@@ -0,0 +1,24 @@
+`extern (C++)` construction, destruction, operators and other mangling improvements
+
+Many improvements have been made to the `extern(C++)` experience in this release cycle.
+
+Mangling improvements include:
+* Constructor/destructor mangling matches C++
+* Compatible D style operators now mangle as C++ operators
+* `nullptr_t` mangles correctly
+* Also various mangling bugs have been fixed
+
+`extern(C++)` APIs that use `nullptr_t` can use `typeof(null)` on the D side:
+---
+alias nullptr_t = typeof(null);
+extern (C++) void fun(nullptr_t);
```

Comment:
Maybe use `extern(C++)` for the sake of consistency? (also applies to some latter usages)